packages/@aws-cdk/aws-cloudfront/package.json

```diff
@@ -92,6 +92,7 @@
   "stability": "experimental",
   "awslint": {
     "exclude": [
+      "props-physical-name:@aws-cdk/aws-cloudfront.OriginAccessIdentityProps",
```

Comment:
why?

Reply:
Because there is no physical name for the constructor but awslint always was a physical name in the props